Build fail with vs 2019

According to README, `On Windows, VS 2017 Build Tools or later should be sufficient.`, users should be able to build oeedger-cpp with vs2019. However, based on my test, the build succeeded with vs2017 but failed with vs2019 on the same machine.

The log is as follows.
```
[10/244] Performing build step for 'oeedger8r-cpp'
FAILED: tools/oeedger8r-cpp-prefix/src/oeedger8r-cpp-stamp/oeedger8r-cpp-build tools/oeedger8r-cpp/oeedger8r.exe
cmd.exe /C "cd /D C:\Users\mingwei\oe-ms\build\x64-2019\tools\oeedger8r-cpp && "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" --build ."
[1/70] Building CXX object CMakeFiles\oeedger8r.dir\lexer.cpp.obj
[2/70] Building CXX object CMakeFiles\oeedger8r.dir\parser.cpp.obj
C:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\parser.cpp(21): warning C4996: 'fopen': This function or variable may be unsafe. Consider using fopen_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
[3/70] Building CXX object test\virtual\CMakeFiles\oeedger8r_test_enclave.dir\enclave.cpp.obj
C:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\test\virtual\enclave.cpp(46): warning C4273: 'wcslen': inconsistent dll linkage
C:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt\corecrt_wstring.h(137): note: see previous definition of 'wcslen'
[4/70] Linking CXX static library test\virtual\oeedger8r_test_enclave.lib
[5/70] Building CXX object test\virtual\CMakeFiles\oeedger8r_test_host.dir\host.cpp.obj
FAILED: test/virtual/CMakeFiles/oeedger8r_test_host.dir/host.cpp.obj
"C:\PROGRA~2\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.25.28610\bin\Hostx64\x64\cl.exe"  /nologo /TP  -IC:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\test\virtual\include /DWIN32 /D_WINDOWS /W3 /GR /EHsc /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /Fotest\virtual\CMakeFiles\oeedger8r_test_host.dir\host.cpp.obj /Fdtest\virtual\CMakeFiles\oeedger8r_test_host.dir\oeedger8r_test_host.pdb /FS -c C:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\test\virtual\host.cpp
C:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\test\virtual\host.cpp(91): error C2039: 'string': is not a member of 'std'
C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.25.28610\include\map(25): note: see declaration of 'std'
C:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\test\virtual\host.cpp(91): error C2065: 'string': undeclared identifier
C:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\test\virtual\host.cpp(91): error C2146: syntax error: missing ';' before identifier 'path_with_ext'
C:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\test\virtual\host.cpp(91): error C2065: 'path_with_ext': undeclared identifier
C:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\test\virtual\host.cpp(91): error C2039: 'string': is not a member of 'std'
C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.25.28610\include\map(25): note: see declaration of 'std'
C:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\test\virtual\host.cpp(91): error C3861: 'string': identifier not found
C:\Users\mingwei\oe-ms\tools\oeedger8r-cpp\test\virtual\host.cpp(94): error C2065: 'path_with_ext': undeclared identifier
[6/70] Building CXX object CMakeFiles\oeedger8r.dir\main.cpp.obj
ninja: build stopped: subcommand failed.
ninja: build stopped: subcommand failed.
```

--- a/test/virtual/host.cpp
+++ b/test/virtual/host.cpp
@@ -9,6 +9,7 @@
 
 #include <openenclave/edger8r/host.h>
 #include <mutex>
+#include <string>
 
 #include "enclave_impl.h"
 

